src/components/atoms/ActionButton.js

```diff
@@ -15,7 +15,8 @@ export function ActionButton(props) {
           : "rounded-md"
       } shadow-lg ${
         props.invert
-          ? "bg-gray-light text-text-gray-dk border border-gray-md hover:bg-bg-gray-dk hover:text-white focus:bg-bg-gray-dk focus:text-white"
+          ? "bg-white text-text-gray-dk border border-gray-md hover:bg-bg-gray-dk hover:text-white focus:bg-bg-gray-dk focus:text-white" +
+            props.classInvert
           : props.className
```

Comment:
Is there a need to identify another property for class names just for invert?
Could you not just always include `props.className` (rather than only include it when not `invert`?

If it does really need to be added, the property needs to be defined in `propTypes`.

Reply:
Agree with shaun